hyperlink/test/test_decoded_url.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
```

Comment:
I am very anti `unicode_literals` because of the trouble it causes with filenames, for example.  I don't expect you to remove it but I did want to mention it!